### ISSUE TITLE
How to use Wwan and 7455 modem together?

### ISSUE THREAD

**Issue by Oleg23011977** [Ysurac/openmptcprouter#2904]
How to use Wwan (Huawei B818-263) and 7455 modem together?

## How to use Wwan and 7455 modem together?

Is it possible to make sure that traffic is always through Wan + LTe??
## Current Behavior

##Specifications

   - OpenMPTCProuter version: 
   - OpenMPTCProuter VPS version: 
   - OpenMPTCProuter VPS provider:Timeweb
   - OpenMPTCProuter platform: Em7455, Em12.Huawei B818-263

**Comment by Oleg23011977** [comment]
At the moment, when using Wan (Master), all traffic goes through it, ignoring the LTE module. If the Wan connection is interrupted, the file download is suspended, after a while it resumes via LTE, but when the wired connection is restored, the speed goes through the wire at a minimum. As if MPTCP is using LTE as (Master).
if you do Speedtest, then I observe that the speed is high, and goes through Wan, after some time (3-7 minutes) the file may continue to load quickly

**Comment by Ysurac** [comment]
What is the version of OpenMTPCProuter used ?
Can you take a screenshort of System->OpenMPTCProuter, Status tab ?

**Comment by Oleg23011977** [comment]
v0.59.2alpha-5.4 r0+16819-1b6e9b3f64
![MpTCP](https://github.com/Ysurac/openmptcprouter/assets/127437667/ff41a1b2-a1fe-4c1c-909f-f63245a61534)

**Comment by Oleg23011977** [comment]
Wan не принимает участие в загрузке. Хотя он активен.
![MpTCP2](https://github.com/Ysurac/openmptcprouter/assets/127437667/a2b9e944-60a9-4bfd-b5df-1a1702f4ffb4)

**Comment by Oleg23011977** [comment]
Later I will order another server and check.
Today, under heavy load, I received an error.
![MpTCP3](https://github.com/Ysurac/openmptcprouter/assets/127437667/06b4e319-f880-4bcd-bccb-a0c1120400cc)

**Comment by Ysurac** [comment]
You need to set SQM download and upload speed.

**Comment by Oleg23011977** [comment]
Pardon my ignorance.
There is an article where it is described.

I tried to enter numbers.

But I can't find the right values.
Therefore, I doubt that I am pointing this out in the right place.

And rented another server. increasing CPU=2cor 3.3
Mem 2g.

As long as I do not connect the Wan wire, the device works correctly and sums up the traffic of 2LTe modules.

But when I connect only Wan , I already start having problems.

Problem solved.

Wan very large Ping > 80 -120 Ms.

To check, I made another Wan channel, everything works as it should.
![MpTCP6](https://github.com/Ysurac/openmptcprouter/assets/127437667/5d512014-e259-45a8-927b-363c9ea3b1d4)
![MpTCP5](https://github.com/Ysurac/openmptcprouter/assets/127437667/7bdd685f-84bd-40f0-a1f8-ac9163e3d61c)
![MpTCP4](https://github.com/Ysurac/openmptcprouter/assets/127437667/8f151129-a543-4253-b08d-e53edb6ee038)
